.github/actions/is-release-week-v1/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import core from '@actions/core'
+import { getWeekNumber, isReleaseWeek } from './utils'
+
+async function main() {
+  try {
+    const oddWeek = core.getInput('oddWeek').toLocaleLowerCase() === 'true'
```

Comment:
Need to fail the action if `oddWeek` was not supplied: 

```suggestion
    const oddWeek = core.getInput('oddWeek', { required: true }).toLowerCase()   
```

To be safe we should also check the value is what we expect: 

```ts
const oddWeek = core.getInput('oddWeek', { required: true }).toLowerCase()

if (oddWeek !== 'true' || oddWeek !== 'false') { 
	core.setFailed('`oddWeek` must be "true" or "false"')
	return
}

const isOddWeek = oddWeek === 'true'

// ... 

```